libs/dtm/dtm_recovery.py

```diff
@@ -153,7 +152,7 @@ def perform_ops(self, workload_info: list, queue, skipread: bool = True,
                                            skip_write=True,
                                            skip_read=skipread,
                                            validate=validate,
-                                           log_file_prefix=f"read_workload_{workload['obj_size']}b",
```

Comment:
Please keep b in log file prefix.

Reply:
this is adding extra b in logs, if we use workload list from 0b-512Mb. 
For other workload lists, do we have to add it here for log file?